indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -68,6 +69,7 @@ def _process_POST(self):
             create_personal_data_fields(regform)
             db.session.add(regform)
             db.session.flush()
+            signals.registration_form_created.send(regform)
```

Comment:
We don't import all signals in `indico.core.signals` so you need `signals.event.registration_form_created` here.